edge/test/integration/device/device_test.go

```diff
@@ -31,6 +32,37 @@ import (
 	. "github.com/onsi/gomega"
 )
 
+//getting Devicestate from subscribed MQTT topic
```

Comment:
Comment should be started with "Devicestate ..."

Reply:
ok